packages/api/src/index.ts

```diff
@@ -10,3 +9,4 @@ export * from "./queries/get-persoon-data";
 export * from "./queries/get-burger-profiel";
 export * from "./queries/get-zaak";
 export * from "./queries/get-bewoner-aantal.ts";
+export * from "./queries/get-erfpacht-contracten";
```

Comment:
Is Erfpacht iets wat in de nl-portal-frontend-libraries moet? Of in de dh-nl-portal-frontend?

Reply:
Dat klopt wel maar de api packages is nu wel de enige die een codegen heeft. Anders zouden we een deel moeten verplaatsen naar /app en datzelfde deel dan ook naar DH